.changeset/eleven-women-glow.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/tree": minor
+"__section": tree
+---
+Renames object in SchemaFactoryAlpha to objectAlpha
```

Comment:
I think @Josmithr is working on getting some guidance on the subject, but I think we want to phrase changesets such that they read well in the release nodes when someone is looking for what changed in the release, and are easy to skim to see if its relevant to your code. For example:

```suggestion
SchemaFactoryAlpha.object was renamed to SchemaFactoryAlpha.objectAlpha
```